gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStorageStatistics.java

```diff
@@ -86,6 +88,13 @@ public GhfsStorageStatistics() {
         maximums.put(getMaxKey(symbol), new AtomicLong(0));
         means.put(getMeanKey(symbol), new MeanStatistic());
       }
+
+      if (opType.getType() == StatisticTypeEnum.TYPE_DURATION_TOTAL) {
```

Comment:
nit: may be the three lines can be move to the previous if block and get rid of the the three line code duplication.